app/controllers/monologue/posts_controller.rb

```diff
@@ -17,5 +17,6 @@ def show
 
   def feed
     @posts = Monologue::Post.published.limit(25)
+    render 'feed', layout: false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.